openfisca_uk/variables/finance/benefit/benefit.py

```diff
@@ -262,13 +262,17 @@ def formula(person, period, parameters):
 
 
 class claims_legacy_benefits(Variable):
-    value_type = float
+    value_type = bool
     entity = BenUnit
     label = u"Whether this family is imputed to claim legacy benefits over Universal Credit"
     definition_period = YEAR
 
     def formula(benunit, period, parameters):
         return (
-            random(benunit)
+            benunit.value_from_first_person(
```

Comment:
Please add a comment - is this what we discussed yesterday about assigning all benefit units in the household the same legacy benefit status?

Reply:
It is, yes